Mention color in types guide and reference (#171)
xodio/xod-docs: docs/reference/data-types/README.md

    </tr>
  </tbody>
</table>

### Color literals

> Values for `xod/color/color` type can be entered in `#RRGGBB` format, where `RR` is how much Red (using two hexadecimal digits), `GG` is how much Green, and `BB` how much Blue. For example, `#00BFFF` will be displayed as a light blue color.

Reviewer: ```suggestion
Values for `xod/color/color` type can be entered in `#RRGGBB` format. There, `RR` is amount of Red (using two hexadecimal digits), `GG` is amount of Green, and `BB` is amount of Blue.

For example, `#00BFFF` will be displayed as a light blue color.
```
